reV/supply_curve/points.py

```diff
@@ -282,6 +288,18 @@ def __init__(
             assert inclusion_mask.size == len(self._gids), msg
             self._incl_mask = inclusion_mask.copy()
 
+        self._zone_mask = zone_mask
+        if zone_mask is not None:
+            msg = (
+                "Bad zone mask input shape of {} with stated "
+                "resolution of {}".format(zone_mask.shape, resolution)
+            )
+            assert len(zone_mask.shape) == 2, msg
+            assert zone_mask.shape[0] <= resolution, msg
+            assert zone_mask.shape[1] <= resolution, msg
+            assert zone_mask.size == len(self._gids), msg
+            self._zone_mask = zone_mask.copy()
```

Comment:
This copy seems to be intentionally placed, but I can't seem to figure out why. Do you ever modify the values within? Did you run into some sort of issues without the copy? or is this just trying to be future-proof?

Reply:
Ok, the copy here is probably not necessary, but let's not have it hold up the merge. We can drop it later if it becomes a problem